src/PrestaShopBundle/Controller/Admin/Improve/Shipping/PreferencesController.php

```diff
@@ -52,7 +52,7 @@ public function indexAction(Request $request)
         $handlingForm = $this->getHandlingFormHandler()->getForm();
         $carrierOptionsForm = $this->getCarrierOptionsFormHandler()->getForm();
 
-        return $this->renderForm($handlingForm, $carrierOptionsForm, $request);
+        return $this->renderFormFromMetaFilters($handlingForm, $carrierOptionsForm, $request);
```

Comment:
same here, but it was a bad idea to override AbstractController::renderForm from Symfony lol